src/shadowbox/server/server_access_key.ts

```diff
@@ -212,8 +220,11 @@ export class ServerAccessKeyRepository implements AccessKeyRepository {
     for (const accessKey of this.accessKeys) {
       const usageBytes = bytesTransferredById[accessKey.id] || 0;
```

Comment:
```suggestion
      const usageBytes = bytesTransferredById[accessKey.id] ?? 0;
```

For consistency. See below.